### PR TITLE
Updated Demodays Page

### PR DESCRIPTION
**Pull request by thisiskaransgit**
# Pull Request Template

## Description

Updated Demodays page with the new keynote speakers and details

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [X] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [X] My code follows the style guidelines of this project.
- [X] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [X] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

**Comment by Namyalg** [comment]
Thank you for this @thisiskaransgit 
The section for the LFX mentees looks great !

File: website/src/pages/demodays.js [hunk 1]
```diff
@@ -1,72 +1,124 @@
...
+            <h2 className={styles.date}> 🗓️ Coming soon...</h2>
+          </div>
+        </section>
+        <section className={styles.about}>
+          <div className={styles.descript}>
+            <h1 className={styles.head}>Agenda</h1>
+            <p className={styles.headscript}>
+              We will have our Graduted mentees from GSoC(Google summer of code), LFX(Linux
+              Foundation), GSOD(Google season of docs) and Outreachy from the batch 2022 showcasing
+              their work and learnings with the community.
```
Comment (HarshCasper) [inline website/src/pages/demodays.js:34]: - Change to `graduated mentees`
- Keep a space between the terms and their abbreviation

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Here is a high-level review:

- Can we change `demodays` to something more useful? Perhaps, something that signifies the mentees we will host over the years which someone can navigate across by click on respective years (like `2022`) ? 
- For @SanjaySinghRajpoot, I would recommend having a more professional picture in place.
- `Agenda` and `What to expect` can be mentioned in a brief box.
- @Namyalg Do we have the recordings hosted somewhere so that we can link it here, please?
- All of the mentee descriptions should be professional and in third person. Can it be standardized?

**Comment by thisiskaransgit** [comment]
Hey @HarshCasper, sorry for the pretty late update, I resolved the changes and the resolved merge conflicts too, we will plan to implement the reviews that you mentioned, for now, could you kindly merge the PR, I don't want it to be closed 😓 ?

**Pull request closed without merging**